Cubical/Algebra/DistLattice/BigOps.agda

```diff
@@ -107,6 +107,15 @@ module Join (L' : DistLattice ℓ) where
  ≤-⋁Ext = ≤-bigOpExt
 
 
+module JoinMap {L : DistLattice ℓ} {L' : DistLattice ℓ'} (φ : DistLatticeHom L L') where
+  private module L = Join L
+  private module L' = Join L'
+  open BigOpMap (LatticeHom→JoinSemilatticeHom φ)
+
+  pres⋁ : {n : ℕ} (U : FinVec ⟨ L ⟩ n) → φ .fst (L.⋁ U) ≡ L'.⋁ (φ .fst ∘ U)
+  pres⋁ = presBigOp
```

Comment:
It seems weird to have this lemma only for `Join` and not for `Meet`. But this was already the case for other definitions in this file (and I find it pretty hard to see which ones)...

I guess this is part of the general question how to handle the duality between join and meet nicely.

Reply:
I added the analogous case for meets of this one: https://github.com/agda/cubical/pull/1096/commits/ef4ae22842f8d581b2bc80f048ba05387624ed60

I did treat meets and joins symmetrically, but I never ended up using big meets, so a bunch of lemmas are only done for big joins. After all, they are what you need to define a "cover", so arguably the more useful one...